### PR TITLE
xtest: replace license terms by SPDX-Licence tags

### PR DESCRIPTION
**Pull request by etienne-lms**
Replace inline written licence terms with an appropriate SPDX licence identifier tag in xtest source files.

### CODE REVIEW

File: host/xtest/regression_1000.c [hunk 1]
```diff
@@ -1,15 +1,8 @@
+// SPDX-License-Identifier: GPL-2.0
 /*
+ * Copyright (c) 2020, Linaro Limited
```
[the remark this answers is not in the document]
Comment (etienne-lms) [reply host/xtest/regression_1000.c:3]: will remove.

File: host/xtest/xtest_helpers.c [hunk 2]
```diff
@@ -1,14 +1,7 @@
+// SPDX-License-Identifier: GPL-2.0
 /*
+ * Copyright (c) 2020, Linaro Limited
```
[the remark this answers is not in the document]
Comment (etienne-lms) [reply host/xtest/xtest_helpers.c:3]: will remove

File: host/xtest/xtest_main.c [hunk 3]
```diff
@@ -1,15 +1,7 @@
+// SPDX-License-Identifier: GPL-2.0
 /*
- * Copyright (c) 2016, Linaro Limited
+ * Copyright (c) 2016-2020, Linaro Limited
```
[the remark this answers is not in the document]
Comment (etienne-lms) [reply host/xtest/xtest_main.c:3]: will discard this line change

File: host/xtest/xtest_test.h [hunk 4]
```diff
@@ -1,15 +1,7 @@
+/* SPDX-License-Identifier: GPL-2.0 */
 /*
- * Copyright (c) 2016, Linaro Limited
+ * Copyright (c) 2016-2020, Linaro Limited
```
[the remark this answers is not in the document]
Comment (etienne-lms) [reply host/xtest/xtest_test.h:3]: will discard this line change

File: ta/storage2/include/user_ta_header_defines.h [hunk 5]
```diff
@@ -1,3 +1,7 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2020, Linaro Limited
+ */
```
[the remark this answers is not in the document]
Comment (etienne-lms) [reply ta/storage2/include/user_ta_header_defines.h:4]: This was introduced without explicit copyright notice in e3854167c8e9 ("Add xtest 6015 (TA storage separation)") with S-o-b tag from jerome.forissier[AT]linaro.org.
We can hardly define a BSD type distribution license without copyrighting the content subject to the license terms.
Ok to add Linaro copyright here?

File: host/xtest/xtest_helpers.h [hunk 6]
```diff
@@ -1,14 +1,7 @@
+/* SPDX-License-Identifier: GPL-2.0 */
 /*
+ * Copyright (c) 2020, Linaro Limited
```
Comment (etienne-lms) [inline host/xtest/xtest_helpers.h:3]: will remove this one also

File: ta/storage2/include/user_ta_header_defines.h (hunk 5 above)
[the remark this answers is not in the document]
Comment (jforissier) [reply ta/storage2/include/user_ta_header_defines.h:4]: OK sounds reasonable. But note the commit is dated 2016....

File: host/xtest/xtest_helpers.h (hunk 6 above)
Comment (etienne-lms) [reply host/xtest/xtest_helpers.h]: Ok, i'll set to 2016.

File: ta/storage2/include/user_ta_header_defines.h (hunk 5 above)
[the remark this answers is not in the document]
Comment (etienne-lms) [reply ta/storage2/include/user_ta_header_defines.h:4]: ok, i'll set the date to 2016.

**Comment by etienne-lms** [comment]
Comments addressed.

**Comment by etienne-lms** [comment]
Fixup commits squashed.
Tag applied.
Thanks @jforissier for the deep review.

**Comment by jforissier** [comment]
Safe to take in 3.9.0.

**Pull request merged**